docs/src/catalyst-standards/permissionless-auth/auth-header.md

```diff
@@ -1,5 +1,7 @@
 # Permission-less Authentication for Catalyst
 
+<!-- cspell: words catid Fftx -->
```

Comment:
Can we move these words to `project.dic`?

Reply:
`catid` we could, but `Fftx` is just an example encoded value which isn't something we should be matching against in the general case.